<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget {
	background-color:rgb(231, 217, 182);
	border-radius: 20px;
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe MDL2 Assets</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(3, 48, 81)</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Type your email or username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>280</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
padding: 0px 10px;</string>
     </property>
     <property name="placeholderText">
      <string>Your email or username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Type your password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>280</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
padding: 0px 10px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Your password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Don't have an account?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color:rgb(255, 255, 255);
    border-radius: 20px;
}

QPushButton:hover {
    background-color: rgb(208, 208, 208);
}
</string>
     </property>
     <property name="text">
      <string>Sign up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color:rgb(231, 217, 182);
    border-radius: 10px;
}
QPushButton:hover {
    background-color: rgb(116, 116, 116);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Resource/Images/left arrow.png</normaloff>../Resource/Images/left arrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>310</y>
       <width>280</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(255, 127, 67);
    border-radius: 20px;
	color: white;
}

QPushButton:hover {
    background-color:rgb(255, 172, 103);
};
</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
